config/zones.json

```diff
@@ -1151,6 +1151,17 @@
     },
     "timezone": "Asia/Kolkata"
   },
+  "IN-UP": {
```

Comment:
Is any information on capacities available?

Reply:
I couldn't find any capacity info on the page.

Reply:
We'll leave it as an outstanding item in the issue.